From 71e4ffd46dbbb7de1d06edb66f3e1fe0de423586 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Wed, 21 May 2025 13:25:46 +0200
Subject: [PATCH] python/Makefile.am: use absolute path for source directory
 includes as well

Otherwise it would be relative to the build directory,
which works only if the build and source directory are in the
same parent directory, and breaks if they're on different levels.

Upstream-Status: Submitted [by email to fche@redhat.com,wcohen@redhat.com,systemtap@sourceware.org]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 python/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/Makefile.am b/python/Makefile.am
index 9d5cb7bcb..130e90b24 100644
--- a/python/Makefile.am
+++ b/python/Makefile.am
@@ -2,10 +2,10 @@
 
 AUTOMAKE_OPTIONS = subdir-objects
 
-# Note that we have to use 'abs_builddir' here since we change
+# Note that we have to use absolute directores here since we change
 # directories back to the source directory when building the python
 # extension.
-AM_CPPFLAGS = -I$(srcdir)/../includes
+AM_CPPFLAGS = -I$(abs_srcdir)/../includes
 AM_CPPFLAGS += -I$(abs_builddir)/../includes/sys
 
 # Add OE's CFLAGS which contains `-fdebug-prefix-map' options to
-- 
2.39.5

